src/utils/list-users.js

```diff
@@ -10,10 +10,10 @@ export default async function *getOptions(query) {
 
   do {
     const res = await get(`/accounts?page=${current_page}&limit=${limit}&q=${query}`);
-    const accounts = await res.json();
-    yield accounts.data;
+	const {data: accounts, pages: new_pages} = await res.json();
```

Comment:
Indentation messed up. Also, put spaces around curly braces in destructuring, like `{ data: accounts, pages: newPages }` (yeah, and casing)